docs/source/model-registry.rst

```diff
@@ -132,11 +142,14 @@ While the method above creates an empty registered model with no version associa
 .. code-block:: py
 
     client = MlflowClient()
-    result = client.create_model_version(name="sk-learn-random-forest-reg-model",
-                source="mlruns/0/d16076a3ec534311817565e6527539c0/artifacts/sk-model",
-                run_id="d16076a3ec534311817565e6527539c0")
+    run_id = "d16076a3ec534311817565e6527539c0"
+    result = client.create_model_version(
+        name="sk-learn-random-forest-reg-model",
+        source="mlruns/0/{}/artifacts/sklearn-model".format(run_id),
+        run_id=run_id
+    )
```

Comment:
I would leave these as full paths `source` path for clarity rather than python logic to insert run ID